routes/markerSites.js

```diff
@@ -87,7 +87,12 @@ module.exports = function(connection) {
 
     /* Create new marker */
     router.post('/marker/', isAuthenticated, function (req, res, next) {
+
         req.body = JSON.parse(Object.keys(req.body)[0]);
+
+        //If authenticated user is not owner of the marker, return 401 (Unauthorized)
+        if(req.session.user.uid != req.body.uid) return res.send(401);
```

Comment:
Does this mean an admin could not create a marker on behalf of someone else? I don't know if the UI would support it at this point, but we might want to allow the possibility.

Also, if we're assuming usernames are both strings, consider using `!==`.

Reply:
Yup, I put it there to avoid that possibility. I can't see how it'll fit into the UI right now. Should I remove the line?

Reply:
I wouldn't remove it. I'd just tweak it so it doesn't apply to admins. Something like:
```node
// If authenticated user is not owner of the marker or an admin, return 401 (Unauthorized)
if (req.session.user.uid != req.body.uid && !req.session.user.admin) return res.send(401);
```

Reply:
done :)